modules/user_auth/README.md

```diff
@@ -35,6 +35,8 @@ No requirements.
 | backend\_service\_domain | Domain of the backend service | `string` | n/a | yes |
 | cookie\_signing\_secret\_key | Default secret key for signing cookies | `string` | n/a | yes |
 | create\_namespace | Whether to create the auth namespace(defaults to true), otherwise just references the namespace | `bool` | `true` | no |
+| disable\_frontend\_https | Whether frontend URLs should be https, unless your developing locally you should leave the default as is. | `bool` | `false` | no |
```

Comment:
Mind inverting this? Maybe something like `frontend_use_https` with a default of `true`.  The double negative of disabled=false is harder to understand.